.flake8

```diff
@@ -1,3 +1,5 @@
 [flake8]
 max-line-length = 88
 extend-ignore = E203,FI10,FI11,FI12,FI13,FI14,FI15,FI16,FI17,FI18
+per-file-ignores = prompt2model/dataset_transformer/prompt_template.py:E501
+ignore = BLK100, W503
```

Comment:
I think these two can actually be ignored _globally_, not just for this file.

BLK100 can be ignored because we are already running black in a separate action, and W503 seems to be commonly ignored by developers, since it conflicts with another warning in flake8, W504.

Reply:
yeah, makes sense. "ignore" would have ignored globally, but I could just add it to extend-ignore directly instead of creating a new line for it.